src/core/abr/abr_manager.ts

```diff
@@ -58,16 +58,18 @@ export default class ABRManager {
   private _initialBitrates : Partial<Record<IBufferType, number>>;
   private _throttlers : IRepresentationEstimatorsThrottlers;
   private _bandwidthEstimators : Partial<Record<IBufferType, BandwidthEstimator>>;
+  private _lowBufferGapMode : boolean;
 
   /**
    * @param {Object} options
    */
-  constructor(options : IABRManagerArguments) {
+  constructor(options : IABRManagerArguments, lowBufferGapMode: boolean) {
```

Comment:
That looks a little out of place there no ?
Why not in the options, or even giving directly the right configuration in the option